<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormWorkers</class>
 <widget class="QDialog" name="FormWorkers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>541</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>601</width>
    <height>541</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>601</width>
    <height>541</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Staff</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>480</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48, 152, 174)</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>444</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NamelineEdit">
        <property name="styleSheet">
         <string notr="true">   background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(46, 52, 54);
    font: 14px;
    min-width: 10em;
    padding: 6px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;</string>
        </property>
        <property name="text">
         <string>Surname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SurnamelineEdit">
        <property name="styleSheet">
         <string notr="true">   background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(46, 52, 54);
    font: 14px;
    min-width: 10em;
    padding: 6px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;</string>
        </property>
        <property name="text">
         <string>Birthdate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="BirthdatelineEdit">
        <property name="styleSheet">
         <string notr="true">   background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(46, 52, 54);
    font: 14px;
    min-width: 10em;
    padding: 6px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EmaillineEdit">
        <property name="styleSheet">
         <string notr="true">   background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(46, 52, 54);
    font: 14px;
    min-width: 10em;
    padding: 6px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;</string>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StatuslineEdit">
        <property name="styleSheet">
         <string notr="true">   background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(46, 52, 54);
    font: 14px;
    min-width: 10em;
    padding: 6px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;</string>
        </property>
        <property name="text">
         <string>Salary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="SalarydoubleSpinBox">
        <property name="styleSheet">
         <string notr="true">   background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(46, 52, 54);
    font: 14px;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="maximum">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-17</x>
     <y>10</y>
     <width>20</width>
     <height>411</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>511</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="doc.qrc">:/doc/966837.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="doc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FormWorkers</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormWorkers</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
